src/Symfony/Component/Messenger/Bridge/Amqp/Transport/AmqpStamp.php

```diff
@@ -45,7 +46,12 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
-    public static function createFromAmqpEnvelope(\AMQPEnvelope $amqpEnvelope, self $previousStamp = null): self
+    public function isEnvelopeRetried(): bool
```

Comment:
would it be more clear to name this isRetryAttempt as you did in Connection? or just isRetry (in Connection also then)?

Reply:
I agree, and thank you for review!
I renamed it everywhere to `isRetryAttempt`, feel free to take a look once again please.